src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  let personStatus;
+
+  if (isMarried) {
+    personStatus =
+      sex === 'f'
+        ? `${partnerName} is my husband`
+        : `${partnerName} is my wife`;
+  } else {
+    personStatus = 'I am not married';
+  }
```

Comment:
let's avoid nested conditions it looks too cumbersome.

let's follow advice from previous review.

```javascript
const personStatus = sex === 'f' ? 'husband' : 'wife'


return (
// other code

      <div className="Person__partner">
        {!isMarried ? 'I am not married' : `${personName} is my ${personStatus}`}
      </div>
)
```

Reply:
this would look more readable